src/Tribe/Attendees.php

```diff
@@ -575,7 +579,7 @@ private function generate_filtered_list( $event_id ) {
 		$filter_name = "manage_{$this->page_id}_columns";
 		add_filter( $filter_name, [ $this->attendees_table, 'get_columns' ], 15 );
 
-		$items = Tribe__Tickets__Tickets::get_event_attendees( $event_id );
+		$items = 'all' === $event_id ? Tribe__Tickets__Tickets::get_attendees_by_args()['attendees'] : Tribe__Tickets__Tickets::get_event_attendees( $event_id );
```

Comment:
Even though the use of `Tribe__Tickets__Tickets` isn't new here, it might be a good time to use an alias with the class import so that it is easier to read.